README.md

```diff
@@ -1,3 +1,110 @@
-# cordova
+# Apache Cordova
 
-Apache Cordova
+- Mobile apps with HTML, CSS & JS
+- Target multiple platforms with one code base
+- Free and open source
+
+## Repositories
+
+A list of all repositories on GitHub: 
+
+### Platforms
+
+- https://github.com/apache/cordova-android
+- https://github.com/apache/cordova-ios
+- https://github.com/apache/cordova-windows
+- https://github.com/apache/cordova-browser
+- https://github.com/apache/cordova-test-platform
+
+#### Deprecated Platforms
+
+- https://github.com/apache/cordova-blackberry
+- https://github.com/apache/cordova-osx
```

Comment:
cordova-osx and cordova-electron are not deprecated, probably more like "experimental". (didn't check them all, but probably others too)

Reply:
hmm, what to use to decide which ones are officially "deprecated" and which ones are "experimental"?

Reply:
I'd move osx to supported platforms (it has platformAPI implemented).

electron isn't really supported. I'd probably just leave it under deprecated for now. Not a ton of value to add a third category of experimental.

Reply:
moved OSX to normal platforms, added the "experimental" subheadline as Electron is so new

Reply:
If you go to the readme and it says deprecated somewhere then it's deprecated, if not, then it's not deprecated, maybe not experimental neither

Reply:
Following that blackberry is still supported and not deprecated?
https://github.com/apache/cordova-blackberry 
Don't think so :/

Reply:
You have found a bug!